src/local/index.js

```diff
@@ -80,15 +80,17 @@ function startProject(config, projectConf) {
 
     // WS events
   wsClient.on(wsEvents.READY, () => {
-    triggerBigSync(projectConf, _.pick(config, ['debug', 'delete']), () => {
-      fsHelper.watch();
-
-      localLog(
-        text.SYNC_ON_CONNECT,
-        projectConf.hostname, (projectConf.prefersEncrypted) ? 'using' : 'not using',
-        'encryption'
-      );
-    });
+    if (projectConf.triggerBigSync !== false) {
```

Comment:
I'm checking for not `false`. If older config has it `undefined` then this would be true which is the current default behavior. `if (projectConf.triggerBigSync) {` would disable it for all older configs that have it undefined.